jetty-start/src/main/java/org/eclipse/jetty/start/CommandLineBuilder.java

```diff
@@ -180,7 +156,32 @@ public String toString(String delim)
             {
                 buf.append(delim);
             }
-            buf.append(quote(arg));
+            buf.append(arg); // we assume escaping has occurred during addArg
+        }
+
+        return buf.toString();
+    }
+
+    /**
+     * A version of {@link #toString()} where every arg is evaluated for potential {@code '} (single-quote tick) wrapping.
+     *
+     * @param delim the delimiter between args, use {@code ' '} (space) for shell executable command line.
+     * @return the toString but each arg that has spaces is surrounded by {@code '} (single-quote tick)
+     */
+    public String toQuotedString(String delim)
```

Comment:
`delim` is not used and the javadoc doesn't look right.

Reply:
delim is used on line 178, and it's passed in as 2 different values from Main.java

Reply:
Ah,  I got confused by the javadoc.  See comment below now.